src/protocolgame.cpp

```diff
@@ -1561,7 +1561,7 @@ void ProtocolGame::sendBasicData()
 		msg.addByte(spellId);
 	}
 
-	msg.addByte(0x00); // is magic shield active (bool)
+	msg.addByte(player->getVocation()->getMagicShield()); // is magic shield active (bool)
```

Comment:
again, check if condition is active?

Reply:
Btw if a player changes vocation where is this updated?